src/webfaf2/templates/problems/list.html

```diff
@@ -32,6 +32,13 @@
                 {{ render_field(filter_form.source_file_names, placeholder="Source files") }}
                 <p class="text-muted">Comma acts as OR, you can use % as a wildcard, at least one backtrace where all three fields are satisfied must exist to show the problem</p>
               </div>
+              <div>
+                {{ render_field(filter_form.since_version, placeholder="Since version") }}
+                {{ render_field(filter_form.since_release, placeholder="Since release") }}
+                {{ render_field(filter_form.to_version, placeholder="To version") }}
+                {{ render_field(filter_form.to_release, placeholder="To release") }}
+                <p class="text-muted">Version range</p>
```

Comment:
Please also add the new fields to line 17 to make the advanced filters stay open.